[2025-06-03T20:29:48.630+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [queued]>
[2025-06-03T20:29:48.647+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [queued]>
[2025-06-03T20:29:48.648+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-03T20:29:48.668+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-06-01 00:00:00+00:00
[2025-06-03T20:29:48.674+0000] {standard_task_runner.py:60} INFO - Started process 62943 to run task
[2025-06-03T20:29:48.679+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'delete_data_from_table', 'scheduled__2025-06-01T00:00:00+00:00', '--job-id', '374', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmpy_ch6jx8']
[2025-06-03T20:29:48.681+0000] {standard_task_runner.py:88} INFO - Job 374: Subtask delete_data_from_table
[2025-06-03T20:29:48.784+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [running]> on host 4bea2aab752f
[2025-06-03T20:29:48.906+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-01T00:00:00+00:00'
[2025-06-03T20:29:48.909+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2025-06-01' and dag_id = 'dag_with_postgres_operator_v01';
        
[2025-06-03T20:29:48.924+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:48.942+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:48.976+0000] {sql.py:450} INFO - Running statement: 
            delete from dag_runs where dt = '2025-06-01' and dag_id = 'dag_with_postgres_operator_v01';
        , parameters: None
[2025-06-03T20:29:48.987+0000] {sql.py:459} INFO - Rows affected: 0
[2025-06-03T20:29:49.009+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=delete_data_from_table, execution_date=20250601T000000, start_date=20250603T202948, end_date=20250603T202949
[2025-06-03T20:29:49.053+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-03T20:29:49.110+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-05T16:04:18.759+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [queued]>
[2025-06-05T16:04:18.791+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [queued]>
[2025-06-05T16:04:18.793+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-05T16:04:18.858+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-06-01 00:00:00+00:00
[2025-06-05T16:04:18.870+0000] {standard_task_runner.py:60} INFO - Started process 51110 to run task
[2025-06-05T16:04:18.884+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'delete_data_from_table', 'scheduled__2025-06-01T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmpf6kj47gg']
[2025-06-05T16:04:18.887+0000] {standard_task_runner.py:88} INFO - Job 85: Subtask delete_data_from_table
[2025-06-05T16:04:19.295+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table scheduled__2025-06-01T00:00:00+00:00 [running]> on host 14aa2e8b0b7d
[2025-06-05T16:04:19.538+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-01T00:00:00+00:00'
[2025-06-05T16:04:19.548+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2025-06-01' and dag_id = 'dag_with_postgres_operator_v01';
        
[2025-06-05T16:04:19.575+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:04:19.598+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:04:19.674+0000] {sql.py:450} INFO - Running statement: 
            delete from dag_runs where dt = '2025-06-01' and dag_id = 'dag_with_postgres_operator_v01';
        , parameters: None
[2025-06-05T16:04:19.697+0000] {sql.py:459} INFO - Rows affected: 0
[2025-06-05T16:04:19.731+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=delete_data_from_table, execution_date=20250601T000000, start_date=20250605T160418, end_date=20250605T160419
[2025-06-05T16:04:19.801+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T16:04:19.848+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
